pkg/pillar/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG BUILD_PKGS_BASE="git gcc linux-headers libc-dev make linux-pam-dev m4 findut
 # we use the same image in several places
 ARG EVE_ALPINE_IMAGE=lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726
 
-FROM lfedge/eve-dom0-ztools:81a7af28b32c7558e1c4622e60a9e3197075decd as zfs
```

Comment:
This reference is causing build failures.

Reply:
This is still failing:
Dockerfile:13
--------------------
  11 |     ARG EVE_ALPINE_IMAGE=lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726
  12 |     
  13 | >>> FROM lfedge/eve-dom0-ztools:f923d706814267c16715ed2db64f289bc479c8d4 as zfs

Reply:
removed pkg/pillar/Dockerfile from PR, will push afterwards